pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -247,14 +248,25 @@ func (r *ReconcileDirectVolumeMigrationProgress) reportContainerStatus(pvProgres
 	}
 
 	if containerStatus == nil {
-		pvProgress.Status.SetCondition(migapi.Condition{
-			Type:     InvalidPod,
-			Status:   migapi.True,
-			Reason:   NotFound,
-			Category: migapi.Critical,
-			Message: fmt.Sprintf("The spec.podRef %s must reference a `Pod` with container name %s",
-				path.Join(podRef.Namespace, podRef.Name), containerName),
-		})
+		if pod.Status.Phase == kapi.PodFailed {
+			pvProgress.Status.PodPhase = kapi.PodFailed
+			pvProgress.Status.SetCondition(migapi.Condition{
+				Type:     PodFailed,
```

Comment:
why is this a condition an not in `pvProgress.Status.Phase`? 

if this is the condition, how would dvm controller pick this? and what would be the effect on migmigration?